src/app/pages/catalogo-bienes/crud-cuentas/crud-cuentas.component.ts

```diff
@@ -0,0 +1,213 @@
+import { Component, OnInit } from '@angular/core';
+import { TranslateService, LangChangeEvent } from '@ngx-translate/core';
+import { CatalogoElementosHelper } from '../../../helpers/catalogo-elementos/catalogoElementosHelper';
+import { Subgrupo } from '../../../@core/data/models/catalogo/jerarquia';
+import { Nivel_t } from '../../../@core/data/models/catalogo/tipo_nivel';
+import { Catalogo } from '../../../@core/data/models/catalogo/catalogo';
+import { Parametro } from '../../../@core/data/models/configuracion_crud';
+import Swal from 'sweetalert2';
+import { ConfiguracionService } from '../../../@core/data/configuracion.service';
+import { PopUpManager } from '../../../managers/popUpManager';
+
+@Component({
+  selector: 'ngx-crud-cuentas',
+  templateUrl: './crud-cuentas.component.html',
+  styleUrls: ['./crud-cuentas.component.scss'],
+})
+export class CrudCuentasComponent implements OnInit {
+  catalogos: Array<Catalogo>;
+  catalogoId: number;
+  uid_1: Subgrupo;
+  infoCuentas: any[];
+
+  cargando: boolean;
+  valid: boolean;
+  estado_cargado: boolean;
+  cuentasNuevas: any[];
+  cuentasPendientes: any[];
+  puede_editar: boolean;
+  texto_sesion_contable: string;
+  texto_estado: string;
+  modificando_cuentas: boolean;
+  claseOk: boolean;
+
+  private estadoAsignacionContable: Parametro;
+
+  constructor(
+    private translate: TranslateService,
+    private catalogoElementosService: CatalogoElementosHelper,
+    private confService: ConfiguracionService,
+    private pUpManager: PopUpManager,
+  ) {
+    this.translate.onLangChange.subscribe((event: LangChangeEvent) => { });
+    this.cargando = true;
+    this.puede_editar = false;
+    this.modificando_cuentas = false;
+    this.catalogos = new Array<Catalogo>();
+    this.catalogoId = 0;
+    this.texto_sesion_contable = '';
+    this.texto_estado = '';
+  }
+
+  ngOnInit() {
+    this.loadCatalogos();
+    this.cargaPermisoEdicionCuentas();
+    this.cargaEstadoSesionContable();
+  }
+
+  private cargaPermisoEdicionCuentas() {
+    const accion = this.confService.getAccion('puedeAsignarCuentas');
+    this.puede_editar = accion ? true : false;
```

Comment:
(En general) Cuando se presente este caso, usar simplemente `!!`
```suggestion
    this.puede_editar = !!accion;
```